Makefile

```diff
@@ -56,7 +55,7 @@ tests-full:
 
 .PHONY: mypy
 mypy:
-	mypy $(mypy_flags) ./hathor ./tests
+	mypy -p hathor -p tests
```

Comment:
Are them the ones listed in `pyproject.toml`? In mypy tool?

Reply:
I ported the flags previously on `$(mypy_flags)` to `pyproject.toml`, this makes it so you can just run `mypy foo/bar.py` to run mypy on a single file with the same checks that would be run when checking the whole project, but much faster, which makes it possible to have a much agile flow for fixing typing errors. The use of `-p hathor` instead of `./hathor` makes no difference though, I could have kept `./hathor` but I was testing `-p hathor` and left it there (same for tests).